Tests/UnitTests/Fatras/Kernel/InteractionListTests.cpp

```diff
@@ -74,7 +83,7 @@ struct L0PointLikeProcess {
 };
 
 struct Fixture {
-  Rng rng{23};
+  std::ranlux48 rng{23};
```

Comment:
is this an RNG we use generally? I suggest we stick to as few different ones as possible.

Reply:
All the Fatras code is templated on the generator type. So I think it is a good idea to actually test it with a few different ones and not just the one that we use in the examples. This is a pretty standard generator that also has the benefit that it is not as complicated (with significantly fewer template parameters) as e.g. the Mersenne twister and gives e.g. much shorter compile error messages.